packages/vue-query/src/useBaseQuery.ts

```diff
@@ -68,15 +67,7 @@ export function useBaseQuery<
   const client = queryClient || useQueryClient()
 
   const defaultedOptions = computed(() => {
-    const defaulted = client.defaultQueryOptions(
```

Comment:
This change causes causes the need to add `any` later in this file, which is not a good thing.
If you get ts errors in this line for some reason, please add `ts-expect-error` in this place and i will take a look at that later on

Reply:
The error is actually coming from eslint, not tsc.

I fixed here now by switching the `as` to  a `:` (as you can see), but let me know if you prefer me to revert back to `as` and use a `// eslint-ignore` comment to get this through eslint

Reply:
this looks better than before imo 👍